packages/eventual-send/test/test-await-no-deep-stacks.js

```diff
@@ -0,0 +1,47 @@
+// This file is not very useful as an
+// automated test. Rather, its purpose is just to run it to see what a
+// deep stack looks like.
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava';
+
+test('no-deep-stacks then', t => {
+  let r;
+  const p = new Promise(res => (r = res));
+  const q = (async () => assert.equal(await ((await p) + 1), 22))();
+  r(33);
+  return (async () => {
+    try {
+      await q;
+    } catch (reason) {
+      t.assert(reason instanceof Error);
+      console.log('expected failure', reason);
```

Comment:
`t.log` is useful here. Ava will show such logs as part of the report, only if fail or verbose. Normal logs get disassociated.